src/ussd_registration.js

```diff
@@ -198,25 +201,39 @@ go.app = function() {
                         return $(get_error_text(name));
                     }
                 },
-                next: 'state_msisdn_mother'
+                next: 'state_msisdn_household'
             });
         });
 
         // FreeText st-3A
-        self.add('state_msisdn_mother', function(name) {
+        self.add('state_msisdn_household', function(name) {
             return new FreeText(name, {
-                question: $(questions[name]),
+                question: $(questions[name]).context({
```

Comment:
Please just comment as to where this value is coming from to explain the complexity